tempesta_fw/t/unit/tfw_fuzzer.c

```diff
@@ -261,17 +274,32 @@ __add_field(char **p, char *end, int t, int n)
 
 		return r.inval;
 	} else {
-		if (n % 2) {
-			add_rand_string(p, end, n * 256,
+		char *v = *p;
+		int len = n * 256;
+		int r;
+
+		if (n % INVALID_FIELD_PERIOD ||
+		    is_only_valid) {
```

Comment:
Please, move '{' to next line. Otherwise, in such multi-line conditions, the condition code mingles with the body code.